src/mpi/coll/ibarrier/ibarrier_tsp_auto.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpiimpl.h"
+
+/* sched version of CVAR and json based collective selection. Meant only for gentran scheduler */
+int MPIR_TSP_Ibarrier_sched_intra_tsp_auto(MPIR_Comm * comm, MPIR_TSP_sched_t sched)
+{
+    int mpi_errno = MPI_SUCCESS;
+
+    MPIR_Csel_coll_sig_s coll_sig = {
+        .coll_type = MPIR_CSEL_COLL_TYPE__IBARRIER,
+        .comm_ptr = comm,
+    };
+    MPII_Csel_container_s *cnt;
+    void *recvbuf = NULL;
+
+    MPIR_Assert(comm->comm_kind == MPIR_COMM_KIND__INTRACOMM);
+
+#if 0
```

Comment:
Essentially, this whole function is commented out. I think it would be better to leave this one as an empty function and when the codes back when it is ready.

Reply:
@zhenggb72

Reply:
fixed